docs/operating-eck/air-gapped.asciidoc

```diff
@@ -33,6 +33,14 @@ To make use of your mirrored images you can either set the image for each applic
 
 
 
+[float]
+[id="{p}-use-internal-operator-image"]
+== Use an internal image of the ECK operator
```

Comment:
Should call it a `mirrored` image instead, not sure. Internal sounds ambiguous.

Reply:
👍